boards/shields/huatian_mt9m114/huatian_mt9m114.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	chosen {
+		zephyr,camera = &nxp_csi;
+	};
+};
+
+&zephyr_cam_i2c {
```

Comment:
I understand this comes from #69810 but this too sounds incorrect? This should be something like `pfc_connector_i2c` once you have properly defined said connector and the appropriate mapping for whichever iMX boards have such connector.